rudder-web/src/main/scala/bootstrap/liftweb/AppConfigAuth.scala

```diff
@@ -134,6 +186,27 @@ class AppConfigAuth extends Loggable {
     }
   }
 
+  @Bean def fileAuthenticationProvider : AuthenticationProvider = {
```

Comment:
No, it is still defined in rudder-web/src/main/resources/applicationContext-security-auth-ldap.xml